mindsdb/integrations/handlers/db2_handler/db2_handler.py

```diff
@@ -61,12 +62,20 @@ def connect(self) -> ibm_db_dbi.Connection:
         # Mandatory connection parameters.
         if not all(key in self.connection_data for key in ['host', 'user', 'password', 'database']):
             raise ValueError('Required parameters (host, user, password, database) must be provided.')
+        cloud = 'databases.appdomain.cloud' in self.connection_data['host']
+        if cloud:
+            connection_string = f"DATABASE={self.connection_data['database']};HOSTNAME={self.connection_data['host']};PORT={self.connection_data['port']};PROTOCOL=TCPIP;UID={self.connection_data['user']};PWD={self.connection_data['password']};SECURITY=SSL;"
 
-        connection_string = f"DRIVER={'IBM DB2 ODBC DRIVER'};DATABASE={self.connection_data['database']};HOST={self.connection_data['host']};PROTOCOL=TCPIP;UID={self.connection_data['user']};PWD={self.connection_data['password']};"
+            if 'port' in self.connection_data:
+                connection_string += f"PORT={self.connection_data['port']};"
+                connection_string += "SSLSERVERCERTIFICATE=;"
```

Comment:
is there a link between port and SSLSERVERCERTIFICATE? has it be set only if port is defined?